### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iamharshit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Srichandra [srichandra](https://github.com/srichandra)
 * Joseph Nii Lante Lamptey [niiwade](https://github.com/niiwade)
 * Utkarsh Gupta [khalibartan](https://github.com/khalibartan)
+* Harshit Agarwal[iamharshit](https://github.com/iamharshit)
```
Comment (raivivek) [inline README.md]: Please put a space after your name.

**Comment by hargup** [comment]
@iamharshit there are merge conflict on your branch. It is a great time to learn what they are and how to fix them, you have got the resources on your hand. In case you get stuck or confused let us know.

**Comment by raivivek** [comment]
Bump @iamharshit

**Comment by proffapt** [comment]
Irrelevant now, closing.

**Pull request closed without merging**